app/controllers/simulator_controller.rb

```diff
@@ -12,6 +12,10 @@ def self.policy_class
     ProjectPolicy
   end
 
+  def mobile_disabled
+    render layout: 'application'
```

Comment:
Why not just use the double-quoted strings as codeclimate recommends?

Reply:
Moved to [Slack](https://circuitverse-team.slack.com/archives/CQFAC5L7P/p1578072012001100)